state/network.go

```diff
@@ -3,29 +3,27 @@ package state
 import "github.com/ChainSafe/gossamer/p2p"
 
 type networkState struct {
-	peer *p2p.Service
+	p2p *p2p.Service
 }
 
 func NewNetworkState() *networkState {
 	return &networkState{
-		peer: &p2p.Service{},
+		// TODO: pass p2p service instance to network state
```

Comment:
addressed in #488

Reply:
After a discussion with David, passing an instance of p2p service to network state might not be the route we want to take with network state. Updated issue #488.